README.md

```diff
@@ -29,100 +30,32 @@ Authsignal.setup do |config|
 end
 ```
 
-## Usage
-
-Authsignal's server side signal API has four main calls `track_action`, `get_action`, `get_user`, `identify`
+You can find your `api_secret_key` in the [Authsignal Portal](https://portal.authsignal.com/organisations/tenants/api).
 
-These examples assume that the SDK is being called from a Ruby on Rails app, adapt depending on your server framework.
+You must specify the correct `baseUrl` for your tenant's region.
 
-### Track Action
-The track action call is the main api call to send actions to authsignal, the default decision is to `ALLOW` actions, this allows you to call track action as a means to keep an audit trail of your user activity.
+| Region      | Base URL                            |
+| ----------- | ----------------------------------- |
+| US (Oregon) | https://signal.authsignal.com/v1    |
+| AU (Sydney) | https://au-signal.authsignal.com/v1 |
```

Comment:
It's `au.` as opposed to `au-` https://docs.authsignal.com/api/server-api#region-selection

It's confusing because the challenge api uses `-`